### PR TITLE
[CFU] Add eyes tests for summary view, second try

### PR DESCRIPTION
**Pull request by rshipp**
Second try for #51414, reverted in #51470. 

-----

Add some feature tests for the summary view on a few Free Response levels, so we can collect visual baselines in advance of refactoring.

Since I just want the visual diffs, I didn't have any specific test cases in mind; open to feedback on whether I should add some `verify` steps or something to these.

Ran locally:
image

image

### CODE REVIEW

**Review by hannahbergam — COMMENTED** [review]
Okay flipping back and forth between this and the first attempt- this is mostly just making sure those music levels stay in allthethings, yes?

**Comment by rshipp** [comment]
Yeah, the only difference should be in the `allthethings` script, and my process there was to pull latest staging, `rake seed:default`, then add the levels I wanted to the script. 

image

**Review by hannahbergam — APPROVED** [review]
✅

**Comment by rshipp** [comment]
ci took way too long, so i'll merge this tomorrow.

File: dashboard/test/ui/features/teacher_tools/level_summary.feature [hunk 1]
```diff
@@ -0,0 +1,32 @@
+@eyes
+Feature: Level summary
+
+Scenario: Free Response level 1
```
Comment (bethanyaconnor) [inline dashboard/test/ui/features/teacher_tools/level_summary.feature:4]: nit: can you add a comment to each of these about what these levels are specifically testing? definitely can be done as a follow up so you don't have to run this through Drone again 😅

**Review by bethanyaconnor — APPROVED** [review]

File: dashboard/test/ui/features/teacher_tools/level_summary.feature (hunk 1 above)
Comment (rshipp) [reply dashboard/test/ui/features/teacher_tools/level_summary.feature:4]: fair. i'll do a followup if this passes eyes

**Pull request merged**